templates/frontend/components/header.tpl

```diff
@@ -36,49 +34,42 @@
-								{url page="index" router=$smarty.const.ROUTE_PAGE}
-							{else}
-								{url context="index" router=$smarty.const.ROUTE_PAGE}
-							{/if}
-						{/capture}
-						{if $displayPageHeaderLogo && is_array($displayPageHeaderLogo)}
-							<a href="{$homeUrl}" class="is_img">
-								<img src="{$publicFilesDir}/{$displayPageHeaderLogo.uploadName|escape:"url"}" width="{$displayPageHeaderLogo.width|escape}" height="{$displayPageHeaderLogo.height|escape}" {if $displayPageHeaderLogo.altText != ''}alt="{$displayPageHeaderLogo.altText|escape}"{/if} />
-							</a>
-						{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_string($displayPageHeaderTitle)}
-							<a href="{$homeUrl}" class="is_text">{$displayPageHeaderTitle}</a>
-						{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_array($displayPageHeaderTitle)}
-							<a href="{$homeUrl}" class="is_img">
-								<img src="{$publicFilesDir}/{$displayPageHeaderTitle.uploadName|escape:"url"}" alt="{$displayPageHeaderTitle.altText|escape}" width="{$displayPageHeaderTitle.width|escape}" height="{$displayPageHeaderTitle.height|escape}" />
-							</a>
+					<div class="pkp_site_name">
+					{capture assign="homeUrl"}
+						{if $currentContext && $multipleContexts}
+							{url page="index" router=$smarty.const.ROUTE_PAGE}
 						{else}
-							<a href="{$homeUrl}" class="is_img">
-								<img src="{$baseUrl}/templates/images/structure/logo.png" alt="{$applicationName|escape}" title="{$applicationName|escape}" width="180" height="90" />
-							</a>
+							{url context="index" router=$smarty.const.ROUTE_PAGE}
 						{/if}
-					{if $requestedOp == 'index'}
-						</h1>
+					{/capture}
+					{if $displayPageHeaderLogo && is_array($displayPageHeaderLogo)}
+						<a href="{$homeUrl}" class="is_img">
+							<img src="{$publicFilesDir}/{$displayPageHeaderLogo.uploadName|escape:"url"}" width="{$displayPageHeaderLogo.width|escape}" height="{$displayPageHeaderLogo.height|escape}" {if $displayPageHeaderLogo.altText != ''}alt="{$displayPageHeaderLogo.altText|escape}"{/if} />
+						</a>
+					{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_string($displayPageHeaderTitle)}
+						<a href="{$homeUrl}" class="is_text">{$displayPageHeaderTitle}</a>
+					{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_array($displayPageHeaderTitle)}
+						<a href="{$homeUrl}" class="is_img">
+							<img src="{$publicFilesDir}/{$displayPageHeaderTitle.uploadName|escape:"url"}" alt="{$displayPageHeaderTitle.altText|escape}" width="{$displayPageHeaderTitle.width|escape}" height="{$displayPageHeaderTitle.height|escape}" />
+						</a>
 					{else}
-						</div>
+						<a href="{$homeUrl}" class="is_img">
+							<img src="{$baseUrl}/templates/images/structure/logo.png" alt="{$applicationName|escape}" title="{$applicationName|escape}" width="180" height="90" />
+						</a>
+					{/if}
+					</div>
+					{if $requestedPage === 'index'}
```

Comment:
I've just noticed that this doesn't work on the homepage when the root URL is used without `/index`. So this line needs to change to:

```
{if !$requestedPage || $requestedPage === 'index'}
```

Reply:
Fixed this